### ISSUE TITLE
Decide how pull requests are handled, and where to discuss this kind of things

### ISSUE THREAD

**Issue by philippejadin** [agorakit/agorakit#520]
@linc I see you merged my pull request #518, thank you. 
I think it means there is some implicit idea that if one of us make changes, the other may merge it if he agrees that it's a good PR. Do we need to formalize this a bit more? (or not ?). I know there is the option to approve PR, like having another "pair of eyes" on it, but have never used it, should we enable this now ?

Also, this kind of discussion should not be in an issue, probably better to use the discussion tab of github (currently not enabled for this repository), but in this case, I'd prefer to use the english dev & users groups here : https://app.agorakit.org/groups/2014 , in the "eat your own dog food" style.

What do you think ?

**Comment by linc** [comment]
> I think it means there is some implicit idea that if one of us make changes, the other may merge it if he agrees that it's a good PR. Do we need to formalize this a bit more? (or not ?). I know there is the option to approve PR, like having another "pair of eyes" on it, but have never used it, should we enable this now ?

I think it's a great _default_ to leave a PR for someone else to review & merge. However, being only 2 of us to review PRs today, I'm reluctant to make it a _requirement_. If one of us takes a break, it stops all forward progress. I suggest leaving a PR open to see if the other is around to take a look, and using your own judgement on whether & when to go ahead and merge it yourself.

> Also, this kind of discussion should not be in an issue, probably better to use the discussion tab of github (currently not enabled for this repository), but in this case, I'd prefer to use the english dev & users groups here : https://app.agorakit.org/groups/2014 , in the "eat your own dog food" style.

I'm amenable to discussing there. Do you have automatic backups you've verified? I've gotten burned by a community going down and finding the backups didn't work. 🙃

That will inspire me to expand the notifications settings much sooner, because I am rather careful about what triggers emails to me.

**Comment by linc** [comment]
I've also turned on "Projects" for this repo to see if that's a useful tool for organizing work: https://github.com/agorakit/agorakit/projects

**Comment by philippejadin** [comment]
- Let's keep the simple PR review process as it is now, thanks for your advises. (and I'm somewhat surprise that I brought the idea of uneeded process, maybe I'm getting older :-))

- app.agorakit has offsite daily backups for the last 30 days at the hosting company + daily at my homeserver + monthly on a separate computer also at my home. I consider it safe.

- good idea to try the github projects  for the next big task

**Comment by zerline** [comment]
> good idea to try the github projects for the next big task

Do you really want to even increase your dependence on github?

cf https://ploum.net/2023-02-22-leaving-github.html

**Comment by philippejadin** [comment]
I agree, but we can just try the feature and see if it helps us or not.
Later on, we could move to another platform, and try to find feature parity
there.
I don't like to rely on github for our work, but the cost of moving is
somehow high imvho.

**Comment by linc** [comment]
I don't think organizing a project on GitHub makes us any more dependent than filing issues here does. It's really just a different view for issues, anyway.

**Comment by zerline** [comment]
Ok
